### ISSUE TITLE
FATAL:setuid_sandbox_host.cc

### ISSUE THREAD

**Issue by mslinn** [mayeaux/videodownloader#4]
Seems the installation instructions should have mentioned the need for setting up chrome-sandbox properly. I'm running on WSL and get this.

```bash
$ sudo chown root node_modules/electron/dist/chrome-sandbox
$ sudo chmod 4755 node_modules/electron/dist/chrome-sandbox
$ npm start

> electron-quick-start@1.0.0 start /mnt/c/work/experiments/node/videodownloader
> electron .

[1790:0702/081142.951857:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that node_modules/electron/dist/chrome-sandbox is owned by root and has mode 4755.
```

**Comment by mayeaux** [comment]
Sorry seems to be something specific to WSL which I haven't come across. Were you able to get it to work?

**Comment by mslinn** [comment]
No. Gave up.

**Comment by mayeaux** [comment]
Yeah sorry about that I hadn't heard of such a thing before. Sounds like it's new to Electron 5.0+ as well. 

This seems to have the fix in there somewhere: https://github.com/electron/electron/issues/17972 Can't really test it myself as I can't replicate it as I'm on OS X but if you find something that works I'm happy to merge a pull request! :)

**Comment by mslinn** [comment]
I'm no longer listening to this thread. Good luck!

**Comment by ubaid-desynova** [comment]
I found the easiest way is to install electron on Windows then start it from WSL.

Steps:

Make sure you have Node.js & npm installed on your Windows machine (you can remove them afterwards) Open cmd.exe, move to your project directory and run npm install electron --save-dev. This will install the Windows version of the prebuilt Electron binary instead of the Linux one, which would occur if you try to install from WSL. (This is the actual trick) Enter Bash on Ubuntu on Windows, move to your project directory then run ./node_modules/.bin/electron (or use an npm script) to launch your Electron app Though, I'm not sure this is very convenient, it seems to work well.

I hope this will help people encoutering the same issue in the future!

Ref: https://github.com/electron-userland/electron-prebuilt/issues/260

**Comment by mayeaux** [comment]
Thank you @ubaid-desynova ! @mslinn Maybe this will solve your problem?

**Comment by kerwanp** [comment]
Here is an other solution to avoid installing Node and NPM to your host machine:
- Uninstall electron `npm uninstall electron`
- Change npm config platform `npm_config_platform=win32`
- Install electron `npm install electron`
- Unset the environment variable `npm_config_platform`

**Comment by mayeaux** [comment]
Thanks @kerwanp ! I'm gonna leave this issue open just so people can find your solution more easily if they run into this.

**Comment by abdulsalamalmahdi** [comment]
@kerwanp thank you very much I abandoned learning about Elecron becasue of this. thanks to you I am back at it again
